NEWS

```diff
@@ -19,6 +19,16 @@ NEWS - user visible changes				-*- outline -*-
 
 * Changes to the COBOL compiler (cobc) options:
 
+** New option --copy COPYBOOK to load copybooks before parsing files. This
+   option can typically be used to perform replacements without modifying
+   the source code
```

Comment:
or to include COBOL prototypes :-)

Reply:
I added a testcase for it, but indeed, it is not completely supported, but at least, it shows a working example despite the warnings.

Reply:
consider to add -Wno-unfinished in that testcase - and at some point we should fix at least the strange RETURNING.

Reply:
done